projects/chef-infra-server.md

```diff
@@ -33,10 +37,6 @@ Lincoln Baker
   - Github: [lbakerchef](https://github.com/lbakerchef)
   - Slack: @Lincoln Baker
 
-Vinay Satish
-  - Github: [vinay-satish](https://github.com/vinay-satish)
-  - Slack: @Vinay Satish
-
```

Comment:
It might be fine to remove yourself from Approvers since owners get pinged automatically for a review.